app/Http/Controllers/Dormitory/PrintController.php

```diff
@@ -36,6 +38,16 @@ public function index() {
                 "free_pages" => Auth::user()->sumOfActiveFreePages()
             ]);
     }
+    public function noPaper(){
+        $userName = Auth::user()->name;
+        $users = DB::table('roles')->select('email')
+            ->join('role_users', 'roles.id', '=', 'role_users.role_id')
+            ->join('users', 'user_id', '=', 'users.id')
+            ->where('roles.name','internet-admin')
+            ->get();
+        Mail::to($users)->send(new NoPaper($userName));
+        return redirect('/print')->with('message','Email sent');
```

Comment:
You could redirect `back` here. Also, please use a locale (the `__()` function), like at the other functions.